embedded-elasticsearch/src/main/java/com/playtika/test/elasticsearch/ElasticSearchContainerFactory.java

```diff
@@ -15,6 +15,7 @@ static ElasticsearchContainer create(ElasticSearchProperties properties) {
                 .withExposedPorts(properties.httpPort, properties.transportPort)
                 .withEnv("cluster.name", properties.getClusterName())
                 .withEnv("discovery.type", "single-node")
+                .withEnv("xpack.security.enabled", "false")
```

Comment:
SSL is now enabled by default in Elasticsearch 8 resulting in `received plaintext http traffic on an https channel` errors without this option. This also disables authentication which is now also enabled by default.